pyccel/codegen/pipeline.py

```diff
@@ -55,84 +55,61 @@ def execute_pyccel(fname, *,
                    debug         = False,
                    accelerators  = (),
                    output_name   = None,
-                   compiler_export_file = None):
+                   compiler_export_file = None,
+                   is_conda_warnings_disabled = False,
+                   is_conda_warnings_detailed = False):
     """
-    Carries out the main steps required to execute pyccel
+    Pyccel internal.
```

Comment:
This is not very descriptive. I suggest:
```suggestion
    Run Pyccel on the provided code.
```